src/Cardano/Shell/Types.hs

```diff
@@ -76,11 +74,11 @@ data CardanoFeatureInit dependency configuration layer = CardanoFeatureInit
 
 -- | The interface for the running feature, the high-level interface we use for running it.
 data CardanoFeature = CardanoFeature
-    { featureName       :: Text
+    { featureName     :: Text
```

Comment:
Again, like above. Please don't change existing alignment, this is already aligned (by tabs).